lib/discourse_theme/cli.rb

```diff
@@ -205,40 +205,40 @@ def run(args, &block)
 
     private
 
-    def ignored_migrations(theme, dir)
-      return [] unless theme && Dir.exist?(File.join(dir, "migrations"))
+    def skip_migrations(theme, dir)
+      return true unless theme && Dir.exist?(File.join(dir, "migrations"))
 
-      existing_migrations =
+      migrated_migrations =
         theme
           .dig("theme_fields")
           &.filter_map do |theme_field|
-            theme_field["name"] if theme_field["target"] == "migrations"
+            if theme_field["target"] == "migrations" && theme_field["migrated"] == true
+              theme_field["name"]
+            end
           end || []
 
-      new_migrations =
+      pending_migrations =
         Dir["#{dir}/migrations/**/*.js"]
           .reject do |f|
-            existing_migrations.any? do |existing_migration|
+            migrated_migrations.any? do |existing_migration|
               File.basename(f).include?(existing_migration)
             end
           end
           .map { |f| Pathname.new(f).relative_path_from(Pathname.new(dir)).to_s }
 
-      if !new_migrations.empty?
-        options = { "Yes" => :yes, "No" => :no }
+      return true if pending_migrations.empty?
+
+      options = { "No (recommanded)" => :no, "Yes" => :yes }
```

Comment:
I think we shouldn't say an option is recommended here. If you install or update a theme/component that already has migrations, then the right thing to do is to run the migrations, but if we say "no" is recommended will confuse users that aren't very familiar with this tool or migrations.

Perhaps we the prompt message should include a small instruction to help users pick the right choice? Maybe it should say "Select No if you're writing the migration and you haven't finished it yet."

Reply:
Good idea :+1: 

I've updated it to something like this:

```
? Would you like to run the following pending theme migration(s): migrations/settings/0001-rename-settings.js, migrations/settings/0002-rename-settings.js
  Select 'No' if you are in the midst of adding or modifying an existing migration.
 (Press ↑/↓ arrow to move and Enter to select)
‣ No
  Yes
```

Reply:
Perfect, thanks!